src/app/Main/Body/QueryPage/Results/TableResults/TableResult-TableContent.component.tsx

```diff
@@ -1,15 +1,15 @@
 /** @jsx jsx */
 import { css, jsx } from "@emotion/core";
 import { useContext } from "react";
-import { ResultsTableComponent as ResultsTable } from "./TableContent-Table.component";
+import { TableContentTableComponent as ResultsTable } from "./TableContent-Table.component";
 import { ExplanationTextComponent as ExplanationText } from "../../../../../Shared/ExplanationText.component";
 import { TableContext } from "../../../../../Shared/Context/TableContext";
 
 const dataTableStyle = css`
     overflow: auto;
 `;
 
-interface ContentInterface {
+interface TableContentInterface {

```

Comment:
TableContentProps?

Reply:
Yes, it was changed in the commit "Unify names of interfaces"